cloud_storage_handler/api/elixircloud/csh/models.py

```diff
@@ -0,0 +1,37 @@
+"""Model for MinIO Configuration."""
+
+from typing import Annotated
+
+from pydantic import BaseModel, conint, constr
+
+
+class MinioConfig(BaseModel):
+    """Configuration for MinIO.
+
+    Attributes:
+        hostname : The hostname where the MinIO server is running.
+                        Defaults to 'localhost'.
+        port : The port on which the MinIO server is running.
+                    Must be between 1 and 65535. Defaults to 9000.
+        access_key : The access key used for authentication with MinIO.
+                          Defaults to 'minioadmin'.
+        secret_key : The secret key used for authentication with MinIO.
+                          Defaults to 'minioadmin'.
+        bucket_name : The name of the bucket where files are stored.
+                            Must be at least 1 character long. Defaults to 'files'.
```

Comment:
maybe this is a nitpick and @uniqueg could better clarify the standard, but you should only keep one tab after the line break, ie

```python
    Attributes:
        hostname : The hostname where the MinIO server is running.
                        Defaults to 'localhost'.

    Attributes:
        hostname : The hostname where the MinIO server is running.
            Defaults to 'localhost'.
```

Well this is how I do it, this firstly help you put more char per line and also is much easier to handle.